ci/continuous-builder.sh

```diff
@@ -18,7 +18,9 @@ export ALIBOT_ANALYTICS_APP_NAME="continuous-builder.sh"
 MIRROR=${MIRROR:-/build/mirror}
 PACKAGE=${PACKAGE:-AliPhysics}
 
-STATE_CONTEXT=build/$PACKAGE${ALIBUILD_DEFAULTS:+/$ALIBUILD_DEFAULTS}
+# This is the check name. If CHECK_NAME is in the environment, use it. Otherwise
+# default to, e.g., build/AliRoot/release (build/<Package>/<Defaults>)
+STATE_CONTEXT=${CHECK_NAME:=build/$PACKAGE${ALIBUILD_DEFAULTS:+/$ALIBUILD_DEFAULTS}}
```

Comment:
As privately discussed, maybe the best option is to simply rename STATE_CONTEXT to CHECK_NAME, adding a default for it. AFAICT no one depends on $STATE_CONTEXT.